domains/environment_domains/variables.tf

```diff
@@ -4,6 +4,7 @@ variable "resource_group_name" {}
 variable "domains" {}
 variable "environment" {}
 variable "host_name" {}
+variable "null_host_header" { default = false }
```

Comment:
Can we add a description as the name is a bit obscure?
https://developer.hashicorp.com/terraform/language/values/variables#input-variable-documentation